scripts/post-uninstall-rpm.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This check works only for RPMs
```

Comment:
This check was copied from https://github.com/pulsar-edit/pulsar/commit/a47da8e3e0627fe242226fb37ce3586999d69fc9. I'm not sure how they came up with this solution, but it works.